### ISSUE TITLE
Croatia: Lowercase on column: name & sort_name

### ISSUE THREAD

**Issue by marfi** [everypolitician/everypolitician-data#7741]
The Croatian Sabor is one of the few datasets that I have noticed that has all values in name & sort_name in Uppercase. Is there a reason for this? Problems with the encoding? It will be nice if for uniformity we have them in Uppercase first and lowercase rest.

**Comment by tmtmtmtm** [comment]
Looks like it's simply because that's the way it's given in the source: e.g. http://www.sabor.hr/Default.aspx?sec=6172

Looks like it might be possible to extract it from the `title` tag instead, though we'd need to check that that's consistent.

**Comment by tmtmtmtm** [comment]
Yep; looks like that's safe enough. It'll involves a bit of fiddling to set this for the previous term, which was already archived, but it as the old URL is still there, I'm hoping it's safely re-scrape-able.

**Comment by marfi** [comment]
Great!

**Comment by tmtmtmtm** [comment]
Unfortunately morph.io has a huge backlog at the minute, so the scraper has been queued for the last two hours. Once that clears up, we'll be able to see whether this fix works well enough.

**Comment by tmtmtmtm** [comment]
Looks like that worked just fine. In the process of switching to this, though, I noticed that one person had completely disappeared from the official source. After a little research, I discovered that people who leave mid-term get moved to a different page: http://www.sabor.hr/concluded-mandates

I've added that to the scraper as well, but again it seems caught in a big backlog at morph. Should hopefully all flow through at some stage this evening.
